certified-connectors/GMO Sign/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+	"properties": {
+	  "connectionParameters": {
+		"username": {
+		  "type": "securestring",
+		  "uiDefinition": {
+			"displayName": "顧客ID",

```

Comment:
@GMO-GSHD-APIGW please use English strings for all connector files. You can refer to [English language strings guidelines](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1) and to [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance).